app/frontend/styles/_bulk-uploads.scss

```diff
@@ -12,6 +12,11 @@
   border-bottom: 1px solid #b1b4b6;
 }
 
+.no-bottom-border,
+.no-bottom-border > tbody > tr:last-of-type td {
+  border-bottom: none;
+}
+
```

Comment:
What's important here is the use of CSS selectors to target the `td` element. While the `.no-bottom-border` class isn't directly used, it could be useful for future reuse.